handbook/company/pricing-features-table.yml

```diff
@@ -929,7 +929,7 @@
 #  ╩ ╩╩ ╩╩═╝╚╩╝╩ ╩╩╚═╚═╝  ═╩╝╚═╝ ╩ ╚═╝╚═╝ ╩ ╩╚═╝╝╚╝  └─ ╩ ╩ ╩╩╚═╩ ╩─┘
 - industryName: Malware detection (YARA/custom IoCs)  # TODO: consider: technically more than YARA, consider generalizing this and including the concept of comparing known binary hashes and other IoCs (either via live query or in the data lake to compare threat intel feed)
   friendlyName: Scan files for zero days and malware signatures
-  description: Use YARA signatures to report and trigger automations when zero days, malware, or unexpected files are detected on a host.
+  description: Use YARA signatures to report and trigger automations when zero days, malware, or unexpected files are detected on a host.  YARA rules can be deployed remotely and privately.
   documentationUrl: https://fleetdm.com/tables/yara
```

Comment:
```suggestion
  documentationUrl: https://fleetdm.com/guides/remote-yara-rules
```

I think let's point to the guide in the top-level URL. This way, if someone clicks to learn more on the [pricing page](https://fleetdm.com/guides/remote-yara-rules) they'll be taken to the guide:
![Screenshot 2025-01-06 at 5 37 11 PM](https://github.com/user-attachments/assets/2cf6302a-5303-4a8b-b61f-f8a09152c030)

Also, it looks like the guide has a link to the table (old URL) 👍 
![Screenshot 2025-01-06 at 5 38 10 PM](https://github.com/user-attachments/assets/44e7cbf0-a976-4960-8f8c-8a006803a94b)

Reply:
done